services/api/src/resources/environment/resolvers.ts

```diff
@@ -344,21 +344,30 @@ export const addOrUpdateEnvironment: ResolverFn = async (
 
 export const addOrUpdateEnvironmentStorage: ResolverFn = async (
   root,
-  { input },
+  { input: unformattedInput },
   { sqlClient, hasPermission },
 ) => {
   await hasPermission('environment', 'storage');
 
+  const d = new Date();
+  const now = d.toISOString().split('T')[0]+' ' + d.toTimeString().split(' ')[0];
```

Comment:
this seems very hacky, is that the correct way to generate a specific formatted string?

Reply:
Yeah, you're probably right. I've updated and reused a formatter function we use elsewhere.